submissions/Zakkarat/memory-pair-game/app.js

```diff
@@ -22,7 +23,9 @@ function cardFlip(elem) {
   let target = elem.target;
   while (target != cont) {
   if (Array.from(cards).some(card => card == target) && founded.every(elem => elem.src != target.lastElementChild.getElementsByTagName("img")[0].src)) {
```

Comment:
Instead of Array.from() use deconstruction [...var]. this is modern syntax and industry standard today.

Reply:
Done!